### ISSUE TITLE
Skia into Flutter AutoRoller failing on `gclient sync`

### ISSUE THREAD

**Issue by gaaclarke** [flutter/flutter#101718]
The last couple of rolls have failed on `build_and_test_linux_unopt_debug`

autoroller: https://autoroll.skia.org/r/skia-flutter-autoroll
failure: https://cirrus-ci.com/task/4507236234952704
log:
```
[0:00:14] Cloning into '/tmp/clean_engine/src/third_party/_gclient_fontconfig_k10da2hx'...
[0:00:22] warning: You appear to have cloned an empty repository.
[0:00:23] fatal: no matching remote head
[0:00:24] fatal: no matching remote head
[0:00:25] fatal: no matching remote head
[0:00:27] fatal: no matching remote head
----------------------------------------
Error: Command 'git -c core.deltaBaseCacheLimit=2g fetch origin 81c83d510ae3aa75589435ce32a5de05139aacb0 --no-tags' returned non-zero exit status 128 in /tmp/clean_engine/src/third_party/fontconfig
```

**Comment by yusuf-goog** [comment]
The referenced commit (3234e16d2322c936f777d1ef046e689e78b6c52e) seems to now be on the repo https://flutter.googlesource.com/third_party/freetype2/+/refs/heads/master

**Comment by yusuf-goog** [comment]
Kicked off a rerun: https://cirrus-ci.com/task/6234539944050688

**Comment by yusuf-goog** [comment]
Fixed in #101724
